packages/ui-library/src/components/textarea/index.ts

```diff
@@ -186,7 +186,7 @@ export class BlrTextarea extends LitElement {
           ? BlrFormCaptionRenderFunction({
               variant: 'error',
               theme: this.theme,
-              size: this.sizeVariant,
+              sizeVariant: this.sizeVariant,
               message: this.errorMessage,
```

Comment:
We have a unit test failing for the textarea. This is because the unit test expects the textarea hint icon to be visible. We're currently not using the `hasIcon: true` property here so it will be undefined and therefore will not show. The way it currently works for formCaption is that `hasIcon` is an optional property. If it's not provided, it will be hidden. I was wondering if we should make it mandatory? If we do make it mandatory, would this mean that each component that uses formCaption should have the ability to toggle the icon on and off? I think it's something that we need to consider and discuss before we merge anything

Reply:
The idea is, that every component that can display an optional Icon would in the future have an hasIcon prop to toggle if the component has an icon or not. If hasIcon is true, we also will have the icon prop to define which icon is shown. This logic should already implemented this way in the input field text, though the hasIcon prop is named showInputIcon instead (will be fixed with the prop alignments).

We discussed this when we went over all the props with Andelka, Lars and Christian. @ChristianHoffmannS2 maybe you have another opinion now or would this still be the preferred way?